src/layouts/ReviewRequest/components/Comments/CommentsDrawer.tsx

```diff
@@ -156,6 +160,7 @@ export const CommentsDrawer = ({
             <SendCommentForm siteName={siteName} requestId={requestId} />
           </Box>
         </DrawerContent>
+        refetchData={refetchCommentsData}
```

Comment:
this seems to have been misplaced instead of on the `SendCommentForm`

Reply:
Good catch - removed in https://github.com/isomerpages/isomercms-frontend/pull/1133/commits/38bd4010d56b9540ee4799e69569899935692c63!